run-custom-mao.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
+
+# This script needs an machine-api-operator image as an input parameter
+# For example: ./run-custom-mao.sh quay.io/sdasu/machine-api-operator:sdasu-fix
+export SCRIPTS_DIR=$(pwd)
+export CUSTOM_MAO_BUILD=$1
+export KUBECONFIG=ocp/auth/kubeconfig
+REPLACEMENT_TEXT="image: ${CUSTOM_MAO_BUILD}"
+MAO_REPO="raw.githubusercontent.com/sadasu/machine-api-operator"
```

Comment:
This should be configurable either via the CLI or by making the variable overridable?